tests/multi_ptr/multi_ptr_access_members.h

```diff
@@ -0,0 +1,203 @@
+    bool get_decorated_result = false;
+
+    T dereference_value;
+    T dereference_op_value;
+    T get_member_value;
+    T get_raw_member_value;
+    {
+      sycl::buffer<T> dereference_value_buffer(&dereference_value,
+                                               sycl::range<1>(1));
+      sycl::buffer<T> dereference_op_value_buffer(&dereference_op_value,
+                                                  sycl::range<1>(1));
+      sycl::buffer<T> get_member_value_buffer(&get_member_value,
+                                              sycl::range<1>(1));
+      sycl::buffer<T> get_raw_member_value_buffer(&get_raw_member_value,
+                                                  sycl::range<1>(1));
+
+      sycl::buffer<T> val_buffer(&value, sycl::range<1>(1));
+      sycl::buffer<bool, 1> dereference_result_buf(&dereference_result,
+                                                   sycl::range<1>(1));
+      sycl::buffer<bool, 1> dereference_op_result_buf(&dereference_op_result,
+                                                      sycl::range<1>(1));
+      sycl::buffer<bool, 1> get_result_buf(&get_result, sycl::range<1>(1));
+      sycl::buffer<bool, 1> get_raw_result_buf(&get_raw_result,
+                                               sycl::range<1>(1));
+      sycl::buffer<bool, 1> get_decorated_result_buf(&get_decorated_result,
+                                                     sycl::range<1>(1));
+
+      queue.submit([&](sycl::handler &cgh) {
+        auto dereference_value_acc =
+            dereference_value_buffer
+                .template get_access<sycl::access_mode::write>(cgh);
+        auto dereference_op_value_acc =
+            dereference_op_value_buffer
+                .template get_access<sycl::access_mode::write>(cgh);
+        auto get_member_value_acc =
+            get_member_value_buffer
+                .template get_access<sycl::access_mode::write>(cgh);
+        auto get_raw_member_value_acc =
+            get_raw_member_value_buffer
+                .template get_access<sycl::access_mode::write>(cgh);
+
+        auto dereference_result_acc =
+            dereference_result_buf
+                .template get_access<sycl::access_mode::write>(cgh);
+        auto dereference_op_result_acc =
+            dereference_op_result_buf
+                .template get_access<sycl::access_mode::write>(cgh);
+        auto get_result_acc =
+            get_result_buf.template get_access<sycl::access_mode::write>(cgh);
+        auto get_raw_result_acc =
+            get_raw_result_buf.template get_access<sycl::access_mode::write>(
+                cgh);
+        auto get_decorated_result_acc =
+            get_decorated_result_buf
+                .template get_access<sycl::access_mode::write>(cgh);
+
+        auto acc_for_multi_ptr =
+            val_buffer.template get_access<sycl::access_mode::read>(cgh);
+        cgh.single_task([=] {
+          const multi_ptr_t multi_ptr(acc_for_multi_ptr);
```

Comment:
That won't work with all address spaces used here.

Reply:
This comment has been applied in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/346/commits/dec1127b47ad83a085ac90c2eb5735ca0e5da221